pkgs/development/compilers/cudatoolkit/redist/overrides.nix

```diff
@@ -10,60 +12,92 @@ in
 
   libcufile = prev.libcufile.overrideAttrs (oldAttrs: {
     buildInputs = oldAttrs.buildInputs ++ [
-      prev.libcublas
+      prev.libcublas.lib
```

Comment:
We probably want `final` unless it triggers an infinite recursion

Reply:
Fixed in the newest, good catch!